### ISSUE TITLE
Allow .egsinp extension when parsing command line arguments

### ISSUE THREAD

**Issue by mchamberland** [nrc-cnrc/EGSnrc#318]
This is a feature request. When running an application from the command line, it would be nice if we could leave in the .egsinp extension of the input file. For example:

`tutor7pp -i test1.egsinp
`
Right now, we get the following result:

```
***************** Error: 
 input file /Users/Marc/clrp/egs_home/tutor7pp/test1.egsinp.egsinp does not exist 

```

I feel like it should be trivial to test for the existence of the .egsinp extension on the argument, but I couldn't quite get it working straight when I took a shot at it (it seems like the command line arguments get parsed in a few different places, maybe?).

**Comment by ftessier** [comment]
Note: pull request #324 incorrectly refers to this issue, instead of issue #314.

**Comment by ftessier** [comment]
Fixed in pull request #320.
